ray-operator/config/crd/bases/ray.io_rayservices.yaml

```diff
@@ -11471,85 +11471,73 @@ spec:
-                    initKwargs:
-                      additionalProperties:
-                        type: string
-                      type: object
-                    maxConcurrentQueries:
-                      format: int32
-                      type: integer
-                    name:
-                      type: string
-                    numReplicas:
-                      format: int32
-                      type: integer
-                    rayActorOptions:
-                      description: RayActorOptionSpec defines the desired state of
-                        RayActor
+                properties:
+                  importPath:
+                    type: string
+                  runtimeEnv:
+                    type: string
+                  serveConfigs:
+                    items:
+                      description: ServeConfigSpec defines the desired state of RayService
+                        Reference to https://docs.ray.
                       properties:
-                        acceleratorType:
+                        autoscalingConfig:
                           type: string
-                        memory:
+                        gracefulShutdownTimeoutS:
                           format: int32
                           type: integer
-                        numCpus:
-                          type: number
-                        numGpus:
-                          type: number
-                        objectStoreMemory:
+                        gracefulShutdownWaitLoopS:
                           format: int32
                           type: integer
-                        resources:
-                          additionalProperties:
-                            type: string
-                          type: object
-                        runtimeEnv:
-                          additionalProperties:
-                            items:
+                        healthCheckPeriodS:
+                          format: int32
+                          type: integer
+                        healthCheckTimeoutS:
+                          format: int32
+                          type: integer
+                        maxConcurrentQueries:
+                          format: int32
+                          type: integer
+                        name:
+                          type: string
+                        numReplicas:
+                          format: int32
+                          type: integer
+                        rayActorOptions:
+                          description: RayActorOptionSpec defines the desired state
+                            of RayActor
+                          properties:
+                            acceleratorType:
+                              type: string
+                            memory:
+                              format: int32
+                              type: integer
+                            numCpus:
+                              type: number
```

Comment:
Is there any way to enforce that this number should be non-negative?

Reply:
We can add some validation.